packages/account/src/Components/personal-details/personal-details.jsx

```diff
@@ -55,21 +54,11 @@ const PersonalDetails = ({
     const { is_appstore } = React.useContext(PlatformContext);
```

Comment:
we are not removing is_appstore here?

Reply:
I guess it will be removed in one of the PRs that have been created before this